cms/static/cms/js/spec/main_squire.js

```diff
@@ -174,7 +174,7 @@
                 exports: 'XBlock',
                 deps: ['xblock/core']
             },
-            'coffee/src/main': {
```

Comment:
A bit of context, the cms-squire test are failing, and I suppose this shim is the cause.